src/base/error_code.h

```diff
@@ -1703,8 +1703,9 @@
 #define ER_FLASHBACK_EXCEED_MAX_NUM_TRAN_TO_SUMMARY -1335
 #define ER_FLASHBACK_SCHEMA_CHANGED                 -1336
 #define ER_FLASHBACK_LOG_NOT_EXIST                  -1337
+#define ER_FLASHBACK_TIMEOUT                        -1338
```

Comment:
How about using ER_FLASHBACK_EXCEED_THRESHOLD?

Reply:
Initially, it was specified with that name, but it was changed to the current name according to the review.
Since people don't know which THRESHOLD was exceeded, the meaning of the error can become ambiguous.

so, I tried to change it to `EXCEED_TIME_THRESHOLD` but it means TIMEOUT..